core/build/webpack.base.config.ts

```diff
@@ -78,6 +78,7 @@ export default {
       inject: isProd == false
     })
   ],
+  devtool: 'source-map',
```

Comment:
it's longest sourcemap alghorithm that we prev changed to increase build time speed

Reply:
Change it please to other one. Currently it’s disabled at all which makes the debugging almost impossible. It was a big oversaw issue with the latest release :(

Reply:
It’s disabled in prod with my change. You haven’t checked the *.prod.ts config where it’s disabled.

Reply:
Ok, sorry i overseen this